diagnostics/flow_dep_diag/cropping.py

```diff
@@ -0,0 +1,26 @@
+import xarray as xr
+
+file_path = None
+output_path = None
+
+#United States BB provided; replace with region of interest
+#lon in range -180,180
+nla=71.36
+sla=18.91
+wlo=-171.79
+elo=-66.96
+ds = xr.open_dataset(file_path, decode_times=False)
+
+#the diagnostic assumes your grid has a longitudinal range -180,180
+#Shift your grid from 0,360 using the following lines
+ds.coords['lon'] = (ds.coords['lon'] + 180) % 360 - 180 #shift the values
```

Comment:
Test condition before applying the shift.

Reply:
I do not think it needs to test for a condition because if the values are between -180,180 the correct grid units will be retained using this calculation. I will add comments to explain this.